client/src/includes/tabs.js

```diff
@@ -277,10 +271,11 @@ class Tabs {
 
   selectTabByURLHash() {
     if (window.location.hash) {
-      const cleanedHash = window.location.hash.replace(/[^\w\-#]/g, '');
```

Comment:
I don't see a reason why we had to "clean" the hash, i.e. allowing only alphanumeric characters, dashes, and the `#` sign itself. This means the feature doesn't work if the anchored element has an `_`. AFAIK the browser's built-in scroll-to-anchor feature doesn't pose this limitation, so I decided to remove it.